src/themes/teams/components/Label/labelVariables.ts

```diff
@@ -1,13 +1,20 @@
 import { pxToRem } from '../../../../lib'
 
 export default () => {
-  const vars: any = {}
-  vars.circularRadius = pxToRem(9999)
-  vars.padding = `0 ${pxToRem(4)} 0 ${pxToRem(4)}`
-  vars.color = 'rgba(0, 0, 0, 0.6)'
-  vars.backgroundColor = 'rgb(232, 232, 232)'
-  vars.startPaddingLeft = '0px'
-  vars.endPaddingRight = '0px'
-  vars.height = pxToRem(20)
-  return vars
+  const color = 'rgba(0, 0, 0, 0.6)'
+
+  return {
+    circularRadius: pxToRem(9999),
+    padding: `0 ${pxToRem(4)} 0 ${pxToRem(4)}`,
```

Comment:
It feels kind a awkward not having the root with with the variables for the component. I know it is complicating things, but with that we would be consistent with the styles.

Reply:
actually, this is the thing that we explicitly agreed upon - the fact that `variables` should be plain object. The alternative approach (of introducing `root`) would make things much more complex. Would rather vote to leave things as they are at the moment